packages/shared/src/components/modals/LegoReferralCampaignModal.tsx

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable @dailydotdev/daily-dev-eslint-rules/no-custom-color */
```

Comment:
As said on team meet this one has a lot of custom styles due to modal being purposely flashy to attract attention. Should be find because it is all contained in this one file but definitely we need to keep it in check and if this kind of design style gets into more components we can make it standard in our tailwind config.